src/controllers/LearningProvider/cancellationPolicyController.ts

```diff
@@ -37,22 +40,24 @@ export class CancellationPolicyController {
 			)
 
 			if (cancellationPolicy) {
-				res.locals.cancellationPolicy = cancellationPolicy
-				next()
+				req.cancellationPolicy = cancellationPolicy
 			} else {
 				res.sendStatus(404)
 			}
+			next()
```

Comment:
The `next()` should be within the if statement (after line 43) as it was in master